src/js/controller/instream-adapter.js

```diff
@@ -83,18 +83,21 @@ define([
             _instream.on('all', _instreamForward, this);
             _instream.on(events.JWPLAYER_MEDIA_TIME, _instreamTime, this);
             _instream.on(events.JWPLAYER_MEDIA_COMPLETE, _instreamItemNext, this);
+            _instream.on(events.JWPLAYER_MEDIA_BEFORECOMPLETE, function() {
```

Comment:
We can only trigger 'complete' if there is no postroll, or after a postroll is complete.